components/stripe/actions/delete-customer/delete-customer.js

```diff
@@ -0,0 +1,21 @@
+const stripe = require("../../stripe.app.js");
+
+module.exports = {
+  key: "stripe-delete-customer",
+  name: "Delete a Customer",
+  version: "0.0.1",
+  description: "Delete a customer",
+  props: {
+    stripe,
+    customer: {
+      propDefinition: [
+        stripe,
+        "name",
```

Comment:
Shouldn't this take the customer ID? https://stripe.com/docs/api/customers/delete